### PR TITLE
Add images processing

### PR DESCRIPTION
**Pull request by luke-denton-aligent**
@timswalling Please review and merge

### CODE REVIEW

File: tasks/images.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+/* jshint node: true */
+'use strict';
+
+var _ = require('lodash'),
+    imagemin = require('gulp-imagemin'),
+    rename = require('gulp-rename'),
+    sourcemaps = require('gulp-sourcemaps');
```
Comment (timswalling) [inline tasks/images.js]: @luke-denton can you please remove `sourcemaps` - they're won't do anything here.

**Comment by timswalling** [comment]
@luke-denton looks good, but can you please make two minor changes: (1) remove `sourcemaps` (see code comment) and (2) bump the package version to `0.2.0`.

**Comment by luke-denton-aligent** [comment]
@timswalling Updates made

**Comment by timswalling** [comment]
@luke-denton can you please remove `sourcemaps = require('gulp-sourcemaps');` on line 7 of image.js, then we're good to go.

**Comment by luke-denton-aligent** [comment]
@timswalling Done

**Pull request merged**